docs/user-guide/kubernetes.md

```diff
@@ -12,13 +12,74 @@ on your machine, as it is the quickest way to get a local Kubernetes cluster set
 
 2. The `kubectl` binary should be [installed on your workstation](http://kubernetes.io/docs/getting-started-guides/minikube/#download-kubectl).
 
+### Role Based Access Control configuration (Kubernetes 1.6+ only)
+
+Kubernetes introduces [Role Based Access Control (RBAC)](https://kubernetes.io/docs/admin/authorization/) in 1.6+ to allow fine-grained control
+of Kubernetes resources and api.
+
+If your cluster is configured with RBAC, you need to authorize Traefik to use
+kubernetes API using ClusterRole and ClusterRoleBinding resources:
```

Comment:
FWIW, this does not seem to be required on GKE: We're running a Traefik DaemonSet without any extra ServiceAccounts or ClusterRoles defined.

Maybe we should add a note saying something alongs the lines of

_Note that on GKE, this may not be necessary_